tests/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -394,11 +394,104 @@ def test_bobby_profiler_user_workflow_multi_batch(
+    # because setting up the statistical assertions (and the required fixtures with the tolerances) is very laborious.
+    # In the future, additional of the "bootstrap" sampling method should be created using domain builders that generate
+    # a small number of domains (ideally, only one domain) so as to make the building of test fixtures more manageable.
+    row_count_range_rule: dict = rules_configs["row_count_range_rule"]
+    parameter_builders: dict = row_count_range_rule["parameter_builders"]
+    row_count_range_parameter: dict = parameter_builders[0]
+    row_count_range_parameter["sampling_method"] = "bootstrap"
+    rules_configs = {
+        "row_count_range_rule": row_count_range_rule,
+    }
+
+    datasource_name: str = "taxi_pandas"
+    data_connector_name: str = "monthly"
+    data_asset_name: str = "my_reports"
+
+    validator: Validator = data_context.get_validator(
+        datasource_name=datasource_name,
+        data_connector_name=data_connector_name,
+        data_asset_name=data_asset_name,
+        data_connector_query={
+            "index": -1,
+        },
+        create_expectation_suite_with_name=bobby_columnar_table_multi_batch[
+            "test_configuration_oneshot_sampling_method"
+        ]["expectation_suite_name"],
+    )
+
+    profiler: Profiler = Profiler(
+        validator=validator,
+        rules_configs=rules_configs,
+        variables_configs=variables_configs,
+        data_context=data_context,
+    )
+
+    expectation_suite: ExpectationSuite = profiler.profile(
+        expectation_suite_name=bobby_columnar_table_multi_batch[
+            "test_configuration_bootstrap_sampling_method"
+        ]["expectation_suite_name"],
+    )
+    expect_table_row_count_to_be_between_expectation_configuration_kwargs: dict = (
+        expectation_suite.to_json_dict()["expectations"][0]["kwargs"]
+    )
+    min_value = expect_table_row_count_to_be_between_expectation_configuration_kwargs[
+        "min_value"
+    ]
+    max_value = expect_table_row_count_to_be_between_expectation_configuration_kwargs[
+        "max_value"
+    ]
+
+    assert (
+        bobby_columnar_table_multi_batch[
+            "test_configuration_bootstrap_sampling_method"
+        ]["expect_table_row_count_to_be_between_min_value_min_value"]
+        < min_value
+        < bobby_columnar_table_multi_batch[
+            "test_configuration_bootstrap_sampling_method"
+        ]["expect_table_row_count_to_be_between_min_value_max_value"]
+    )
+    assert (
+        bobby_columnar_table_multi_batch[
+            "test_configuration_bootstrap_sampling_method"
+        ]["expect_table_row_count_to_be_between_max_value_min_value"]
+        < max_value
+        < bobby_columnar_table_multi_batch[
+            "test_configuration_bootstrap_sampling_method"
+        ]["expect_table_row_count_to_be_between_max_value_max_value"]
```

Comment:
Should these comparisons be inclusive i.e. `<=`, could the bootstrapped values be the same as the min / max?
```suggestion
    assert (
        bobby_columnar_table_multi_batch[
            "test_configuration_bootstrap_sampling_method"
        ]["expect_table_row_count_to_be_between_min_value_min_value"]
        <= min_value
        <= bobby_columnar_table_multi_batch[
            "test_configuration_bootstrap_sampling_method"
        ]["expect_table_row_count_to_be_between_min_value_max_value"]
    )
    assert (
        bobby_columnar_table_multi_batch[
            "test_configuration_bootstrap_sampling_method"
        ]["expect_table_row_count_to_be_between_max_value_min_value"]
        <= max_value
        <= bobby_columnar_table_multi_batch[
            "test_configuration_bootstrap_sampling_method"
        ]["expect_table_row_count_to_be_between_max_value_max_value"]
```

Reply:
@anthonyburdi Irrespective of the fact that this test will be removed, the new test will also have the "strictly greater" comparison, because these are statistical tests, and the potential equality will be inconsequential (we are comparing outcomes, not probabilities).  Thanks!